lib/roo_on_rails/railties/database.rb

```diff
@@ -0,0 +1,12 @@
+module RooOnRails
+  module Railties
+    class Database < Rails::Railtie
+      initializer 'roo_on_rails.database', after: 'active_record.initialize_database' do |_app|
+        $stderr.puts 'initializer roo_on_rails.database'
+
+        ActiveRecord::Base.configurations[Rails.env]['timeout'] = ENV.fetch('DATABASE_TIMEOUT', 200)
```

Comment:
Does this actually set the statement timeout for Postgres? Or the connection timeout?